tarantool_test.go

```diff
@@ -2830,6 +2830,50 @@ func TestStream_DoWithClosedConn(t *testing.T) {
 	}
 }
 
+func TestConnectionProtocolInfo(t *testing.T) {
+	conn := test_helpers.ConnectWithValidation(t, server, opts)
+
+	isProtocolInfoUnsupported, err := test_helpers.IsTarantoolVersionLess(2, 10, 0)
```

Comment:
It will be better to use conn.IsProtocolVersionSupported() or something like it.